libbpf-rs/src/map.rs

```diff
@@ -455,10 +455,18 @@ impl Map {
 
         let mut out: Vec<u8> = Vec::with_capacity(out_size);
 
+        let mut map_key = key.as_ptr() as *const c_void;
+        if self.key_size() == 0
+            && (matches!(self.map_type(), MapType::Queue)
+                || matches!(self.map_type(), MapType::Stack))
+        {
+            map_key = std::ptr::null();
```

Comment:
Can you please import at top level?
```rust
use std::ptr;
```

Here and elsewhere.

Reply:
seems that it's already import at the top level, I did the change in the code.